mysql-test/suite/innodb/t/import_tablespace_schema_missmatch.test

```diff
@@ -46,7 +46,9 @@ ALTER TABLE t1 DISCARD TABLESPACE;
 
 --move_file $MYSQL_TMP_DIR/t1.ibd $MYSQL_TMP_DIR/alt_dir/$DB/t1.ibd
 
+--disable_warnings
 ALTER TABLE t1 IMPORT TABLESPACE;
+--enable_warnings
```

Comment:
warning is caused by cfg missing. Why the cfg is removed with this PR?

Reply:
It is existing test case, to test import without the cfg file.

Reply:
and the warnings didn't exist before? :thinking:

Reply:
It is upstream test case, MS doesn't allow but PS allows

Reply:
Ok, this testcase is not very clear whether import succeeds or not. IMPORT TABLESPACE errored Can you please add different data in source and destination table and add SELECT (after IMPORT) that shows data is always from source table